vet.rsc.OpenRSC.Launcher.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>vet.rsc.OpenRSC.Launcher.desktop</id>
+    <name>OpenRSC</name>
+    <developer_name>OpenRSC</developer_name>
+    <summary>OpenRSC Client</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>AGPLv3</project_license>
```

Comment:
Buildbot is failing since AGPLv3 is not a license. Where can I see the valid licenses? (Upstream is AGPL v3)

Reply:
https://spdx.org/licenses/